activerecord/CHANGELOG.md

```diff
@@ -1,3 +1,35 @@
+*   Introduces ActiveRecord::Relation#delete_by and ActiveRecord::Relation#destroy_by.
+
+    `destroy_by` allows relation to find all the records matching the condition and perform
+    `destroy_all` on the matched records.
+
+    Example:
+
+        persons = Person.all
+
+        persons.destroy_by(id: [1, 2, 3])
+        persons.destroy_by(name: 'David')
+        persons.destroy_by(name: 'David', rating: 4)
```

Comment:
I think the plural of person is people (at the very least Rails’ inflector says so), so let’s go with that.